.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Test
+
+on: [pull_request, release]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ windows-latest, ubuntu-latest, macos-latest ]
+        python-version: [ "3.10", "3.11" ]
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install Poetry
+        uses: abatilo/actions-poetry@v2.3.0
+
+      - name: Install dependencies
+        run: poetry install
+
+      - name: Run example config
+        run: |
+          # Get example config
+          poetry run python epi_downloader.py --dump-config
+
+          # Try downloading some data
+          poetry run python epi_downloader.py -c example_config.json -o data.csv
```

Comment:
I guess this is connecting to their servers, right? It might be a bit intrusive (and flaky, as it depends on the availability of the service) to include that sort of requests in a CI system. Although it is more work, I'd write a proper integration test mocking the service, so it is totally self-contained.

Reply:
Sorry for the slow response!

I take your point, but I didn't want to get into the realms of writing proper unit tests before we'd had confirmation from the Health-GPS folks that they actually wanted this tool. One advantage of doing it the way I have is that it does also check whether the API has changed on the server's end (as @jamesturner246 suggests, we could also have this check as a cron job), so I would be inclined to leave this check in, even if it's not normally what we'd do. You're right that it needs proper tests on top of this, though.

Not sure what you want to do at this point. I wasn't planning on working on this anymore until I've confirmed with Ali that he's happy with it and the team wants it (I think he's been using it though, which is good!). We could either merge this PR as is or leave it open until I get around to writing proper tests (or at least an integration test that mocks the remote service) or merge it now and polish it later.